src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -114,8 +114,24 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
         "NavBarSchedule__weekday--active": dayIndex === selectedDayIndex,
       });
 
+      const handleKeyPress = (e:React.KeyboardEvent<HTMLLIElement>) => {
+        const enterOrSpace =
+        e.key === "Enter" ||
+        e.key === " " ||
+        e.key === "Spacebar" ||
+        e.which === 13 ||
+        e.which === 32;
+        if (enterOrSpace) {
+          e.preventDefault();
+          setSelectedDayIndex(dayIndex)
+        }
+       }
+
       return (
         <li
+          role="button"
+          tabIndex={0}
+          onKeyPress={(e) => handleKeyPress(e)}
```

Comment:
We shouldn’t need to explicitly handle key presses here if we use the correct base types that naturally support these navigations. Eg. Button within the li